### ISSUE TITLE
Show logs or link to Job Analysis logs in the dashboard

### ISSUE THREAD

**Issue by danil-smirnov** [argoproj/argo-rollouts#3054]
# Summary

When Analysis is run via a Job, logs of the run should be exposed in the dashboard. Alternatively, it could be a link, that is retrieved by the job. Or, at least, it could be a link defined in the Analysis Template statically.

# Use Cases

We trigger our Testkube tests via an Analysis - Job provider. We want to have easy access to the failed tests report via a link or directly in the dashboard. We output that report as our job's logs.

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritize the issues with the most 👍.

**Comment by zachaller** [comment]
This is a very non-trivial thing to add as was brought up here, https://github.com/argoproj/argo-rollouts/issues/2482 this actually also led to one of many reasons that we enabled plugin support, I am not sure what you are doing in your job but it could be more productive to possibly do it as a plugin as well

**Comment by danil-smirnov** [comment]
@zachaller Is plugin from https://github.com/argoproj/argo-rollouts/issues/2482 publicly available? I'm not sure...

Did you consider showing more information for Analysis in the dashboard than just the name?

It would be great if we can click and see metric evaluation result in right the UI - isn't this the UI has been built for?

And for the Analysis Job type it can be logs of the job pod...

**Comment by zachaller** [comment]
Sorry, I see you just want the pod logs that seems reasonable. I was thinking you possibly wanted to return a link from the output of a job. In this case just showing the logs of the pods I think is an easier ask

**Comment by danil-smirnov** [comment]
Yes, and minimal solution would be just to set an optional static link in the job analysis template and show this link as a "See more" button in the UI.

We can point that link to the particular test page in Testkube to make both things connected.

More advanced impovement would be to show logs of the job pod since we can stream the test results into there via a Testkube API call.

**Comment by utsheth-te** [comment]
Are there any updates on this? Maybe a workaround that will not require the use of a plugin?
